src/components/Header.vue

```diff
@@ -53,6 +53,14 @@ export default {
             searchShown: false,
         }
     },
+    computed: {
+        name() {
+            return window.srConfig.BAR_NAME || 'Salt Rim';
+        },
+        description() {
+            return window.srConfig.DESCRIPTION;
```

Comment:
Not sure exactly where I'd set the previous text as default (is there a default config.js somewhere?). 

But wanted to make sure that if a user wanted to omit the description text, the UI still looks good

Reply:
Never mind, spotted where the docker config comes through and set it there too.